nixnet/_session/frames.py

```diff
@@ -22,6 +22,23 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Frame(handle, index, name)
 
+    @property
+    def payld_len_max(self):
+        """int: Returns the maximum payload length of all frames in this session, expressed as bytes (0-254).
+
+        For CAN Stream (Input and Output), this property depends on the XNET
+        Cluster CAN I/O Mode property. If the I/O mode is CAN, this property is
+        8 bytes. If the I/O mode is 'constants.CaNioMode.CANFD' or
+        'constants.CaNioMode.CANFD', this property is 64 bytes.
+
+        For LIN Stream (Input and Output), this property always is 8 bytes. For
+        FlexRay Stream (Input and Output), this property is the same as the XNET
+        Cluster FlexRay Payload Length Maximum property value. For Queued and
+        Single-Point (Input and Output), this is the maximum payload of all
+        frames specified in the List property.
```

Comment:
The comment about Queued and Single-point, is that applying to all protocols?

If so, that should be a separate paragraph.

(the fun of drawing attention to previous work)